packages/expo-font/src/memory.ts

```diff
@@ -1,2 +1,32 @@
-export const loaded: { [name: string]: boolean } = {};
+import ExpoFontLoader from './ExpoFontLoader';
 export const loadPromises: { [name: string]: Promise<void> } = {};
+
+// cache the value on the js side for fast access to the fonts that are loaded
+let _cache: { [name: string]: boolean } = {};
```

Comment:
We try not to use `_` to prefix "private" variables/consts. If you see code doing it, it's probably pretty old 😅 
```suggestion
let cache: Record<string, boolean> = {};
```

Reply:
renamed